src/architecture/transactions.md

```diff
@@ -218,6 +218,26 @@ fn info(&self) -> ::serde_json::Value
 The `info` method returns the useful information (from service developers point
 of view) about the transaction in the JSON format.
```

Comment:
"The `info` method returns the useful information (from service developers point of view) about the transaction in the JSON format." -->
"The `info` method returns useful information (from the service developer's point of view) about the transaction in JSON format."